lfucg-exactions/server/accounts/helper.py

```diff
@@ -39,3 +42,42 @@ def send_lost_username_email(user):
     msg = EmailMultiAlternatives(subject, text_content, from_email, [user.email])
     msg.attach_alternative(html_content, "text/html")
     msg.send()
+
+def send_email_to_supervisors(**kwargs):
```

Comment:
why not specify all the parameters with defaults? send_email_to_supervisors(pk=None, entry=[], group, perm_name)
That way no need to check if its in kwargs. What is the Permission object used for other than getting the model name? Couldn't you just pass in the model name string as a parameter?

Reply:
I didn't think to specify defaults. 
Permissions were more of a "seek-and-find" type of thing. Was thinking we might need it later. 
And yeah definitely on passing as a string.

Reply:
Also just found out what `*args` an `**kwargs` are so I was really bent on using them.